m4_sessions/1708-inning/group_six.md

```diff
@@ -1,53 +1,82 @@
+2. Confront the truth 
+3. Gain elite perspective 
+4. ???
+5. Profit!
+
+### Session Outcomes
 
 Give students the tools to determine for themselves what is important. 
 Worksheet with top five priorities and top five things you don't give a fuck about for personal and professional life
 
 #### Topic Ideas
 * Remove and filter out external pressures and learn to recognize when they affecting your decision making
 
-* The perspective of privilage. Being self aware. Getting worked up over little things compared to real levels of adversity and how it relates to the way we code. Without minifying the personal struggle.
-
 * Not falling into social confitioning traps and understanding the hedonic adaptation and how it affects your hapiness
 
 * Stop wasting your time so you can use it better and feel more fulfilled.
 
 ### Session Outline:
 
-Ice Breaker
-
-Hand out worksheet for priority list and explain
-
-TedTalk
-
-Turn and Talk 2-3 about discussion points
-
-Group talk about those specific discussion points
-
-Topic 1 - Intro
-Topic 1 Group Discussion
-
-Topic 2 - Intro
-Topic 2 Group Discussion
-
-game time
-
-Topic 3 -intro
-Topic 3 Groupd Discussion
-
-Conclusion and Sharing Worksheets(if comfortable)
-
-
+Intro - Will - 5min
+
+Ice Breaker - Adam  - 5min
+
+Hand out worksheet for priority list and explanation - 2min - Lindsay
+
+TedTalk - pt1 - 4min
+
+  Turn and Talk - 5min
+   * How does this idea of not giving a fuck relate to your decision to come to Turing?
+   * How can this mentatlity be hepful for us? Specifically as developers?
+   * What are the dangers of "NFGAF"?
+   
+  Two or Three people share w/ Discussion - 5min
+
+TedTalk - pt2 - 4min
+  
+  Ice-Burg Exercise/Mental Barn Four categories - school/family/friends/extra - 15min
+  * Four Categories
+  * List the top 5 most difficult/annoying/stressful things in those categories
+  * Decide how many "FB's" You have to spend in that category
+  * Decide what you "DGAF" about.
+  * Cross them off with a big black marker
+  
+  Two or Three people share w/ Discussion - 5min
+ 
+TedTalk - pt3 - 4min
+
+  Pair Exercise - 10min - Ben
+    1 Split into pairs. 
+    2 Share a story with your partner about a time you felt obligated and guilty 
+    3 Your partner will help you spin that perspective and decide how these principles apply
+    
+Closing discussion and worksheets - Will - Rest of time
+  * As we discuss start thinking about how to fill out the other side of the worksheet. What are the top five things
+  that are most important to you. What five things will you immediately stop GAF about.
+  * Discuss the hedonic adaptation. How does this relate to what we're talking about. How can we use this perspective
+  to be happier and more fufilled people.
+  * How has guilt, social pressure and obligation negatively affected your decision making in the past. 
+  * What role does our perspective play in what we "GAF" about? Can empathy help "NGAF" in the most positive way?
```

Comment:
And I might add, how might this help you be successful at Turing and in your future job hunt?